obs-browser-source.hpp

```diff
@@ -42,6 +42,9 @@ struct BrowserSource {
 
 	std::string           url;
 	std::string           css;
+    bool                  css_active               = false;
+	std::string           javascript;
+    bool                  javascript_active        = false;
```

Comment:
Style: Spaces have been used here for indentation. Please use a tab instead.

Reply:
Ah sorry, I was under the impression that you want ONLY spaces. It's a while since I saw mixed indentations required. Will fix while removing the CSS flag as requested above.